setup.py

```diff
@@ -19,7 +19,9 @@
       license='LGPLv2+',
       url='https://github.com/kiwitcms/tcms-api',
       python_requires='>=3.6',
-      install_requires=['kerberos'],
+      install_requires=[] + (
+          ["winkerberos"] if sys.platform.startswith("win") else ["kerberos"]
+      ),
```

Comment:
Yes, something like this but you don't need the `[] +` part. You can assign the expression in brackets directly to install_requires.

You also need the same conditional in xmlrpc.py to import the appropriate module depending on the platform. 

Also please squash related commits together.